controller/backup_volume_controller.go

```diff
@@ -314,10 +314,27 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		log.Infof("Found %d backups in the backup target that do not exist in the cluster and need to be pulled", count)
 	}
 	for backupName := range backupsToPull {
+		backupLabelMap := map[string]string{types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode): string(longhorn.AccessModeReadWriteOnce)}
```

Comment:
This should be an empty map, because it will be mutated in the mutator.

Reply:
Modified to empty map. Thanks.